plugins/tweet.py

```diff
@@ -14,6 +14,9 @@
 CONVERTED_IMG = Config.DOWN_PATH + "img.png"
 CONVERTED_STIKR = Config.DOWN_PATH + "sticker.webp"
 
+# idea of short name of celebrities by @ravana
```

Comment:
rula dege bey

Reply:
> rula dege bey

Mera bhai 🤗

Reply:
[Intro]
Honey Singh (Honey Singh)
Badshah... Choot volume 1

[Chorus]
Kenday pechayian pindaan ney teree mari
Saaday lund ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

Kenday pechayian pindaan ney teree mari
Saaday lund ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

[Verse 1]
Aja teri choot maroon
Tere sir se chudne ka bhoot utaroon
Chodne ke baad tujhe jutey maroon
Tere mooh main apna lauda dey key
Moot maroon (yeah)

Kar doon teri fuddi kharab
Merey jesa koi nahin meri buddhi kharab
Terey baad tere post
Bas yahi kaam kaaj mera
Kar doon ga khush tujhe
Lauda ley key naach mera

[Verse 2: Honey Singh & Badshah]
Choda hoga tujh ko hazaroon ney
Mehngay mehngay kamroon mein
Lambi caroon mein
Bollywood key baray baray sitaron ney
Per merey lund jeesa dum naa bicharoon mein

Mein karoon din raat ek
Tu Guzar mere saath ek raat dekh
Terey jesey maine toree ek raat mein
Aazma key dekh ley, dum kitna hai baat mein

[Chorus]
Kenday pechayian pindaan ney teree mari
Saaday lun ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

[Verse 3: Badshah]
Gora badaan tera patli kamar
Solah, satra saal ke umar
Rehti tip top, sunti hip hop
Ley key chaloon tujh ko main candy shop

Ch-ch-choos mera lauda
Aja choos mera lauda
Chaat merey tatte jaise aaloo ka pakoora
Tu hai choot ke rani
Main hoon laudon ka raja
Aja aja mere rani meri bansuri bajaja

Thoda uuper thoda neechey
Zara haat to bataa
Dekh ne dey laal fudi zara jhat to hata
Moon khool apna mera naam to chilaa
Badshah, badshah keh key gaand tu hilaa

[Verse 4: Honey Singh]
Lun-lun-lun utey (Honey Singh)

Lun utey laya appan sanday da tail
Chood chood ker doon mein fudi teri fail
Putt putt mommey tere waday kardoon
Kano sharmawey elay lun phar too

Oey gaal kar to
Main tah bada paad doon
Laa lei barian nu kundi booha bund kar to
Merey lun da waar jiwen chaley talwaar
Aj cheer do mein teri patiala salwar

Maar maar chupey tenu pai gaye si chaskey
Hun mare chika jadoon payaa wich kas ke
Aja lun teh to beja tera pose banawaan
Aj nai kaal nai roj banawaan

Khoono Khoon Hoje kachi jive dul gayi dava
Chap chap marey kasey jatt sari sari raat
Oh mera naa Honey Singh rati batiyaan bujhwaan
Nawi gadeyaan chalawaan naley fudiyaan sujaawaan

[Chorus]
Kenday pechayian pindaan ney teree mari
Saaday lun ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

Kenday pechayian pindaan ney teree mari
Saaday lun ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

[Outro]
Choot
If you can do this hip hop
Here we are
The Choot from Honey Singh
This is Choot exclusively for you
(Honey Singh & Badshah)

Aja teri Choot maroon
Tere sir se chudne ka bhoot utaroon
Chodne ke baad tujhe jutey maroon
Tere mooh main apna lauda dey key
Moot maroon
(Honey Singh & Badshah)

[Chorus]
Kenday pechayian pindaan ney teree mari
Saaday lun ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

Kenday pechayian pindaan ney teree mari
Saaday lun ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

Kenday pechayian pindaan ney teree mari
Saaday lun ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon

Kenday pechayian pindaan ney teree mari
Saaday lun ney we khichee ey tyaari
Fudi teri aj ley kay jaoon
Jey nai liti tey main jatt na kwahoon